kubernetes/kubernetes.go

```diff
@@ -443,3 +450,20 @@ func NewKubernetesJobClient(name string, namespace string) (*KubernetesJobClient
 	}
 	return &KubernetesJobClient{Clientset: clientset, JobName: name, Namespace: namespace}, nil
 }
+
+func getConcurrencyPolicy() batchv1.ConcurrencyPolicy {
+	concurrencyPolicyEnv := helpers.GetEnv("JOBS_CONCURRENCY_POLICY", "Allow")
+	var concurrencyPolicy batchv1.ConcurrencyPolicy
+	switch concurrencyPolicyEnv {
+	case "Allow":
+		concurrencyPolicy = batchv1.AllowConcurrent
+	case "Forbid":
+		concurrencyPolicy = batchv1.ForbidConcurrent
+	case "Replace":
+		concurrencyPolicy = batchv1.ReplaceConcurrent
+	default:
+		fmt.Printf("invalid concurrency policy: %s, defaulting to Allow", concurrencyPolicyEnv)
+		return batchv1.AllowConcurrent
```

Comment:
I'm thinking we should just default to allow in case someone fucks up the env var

Reply:
yeah `allow` is the intuitive default _if_ we're not throwing a panic due to the faulty env variable, but I feel like we should. let's see what others say.